platform/broadcom/sonic-platform-modules-ufispace/debian/sonic-platform-ufispace-s6301-56st.postinst

```diff
@@ -0,0 +1,3 @@
+depmod -a
+systemctl enable pddf-platform-init.service
```

Comment:
For every .postinst file, there should be a corresponding .prerm file which would 'stop' and 'disable' the services enabled in the .postinst file. The effect is particularly seen when .deb pkg installation and removal is tested. 

e.g. for this .postinst file, the .prerm file should have following,
```
# cat broadcom/sonic-platform-modules-ufispace/debian/sonic-platform-ufispace-s9300-32d.prerm
systemctl stop pddf-platform-init.service
systemctl disable pddf-platform-init.service
# 
```

Reply:
@FuzailBrcm I appreciate your reminder. I have updated the .prerm file. Please help me review the code. Thanks.